plugins/package-managers/bazel/src/main/kotlin/MultiBazelModuleRegistryService.kt

```diff
@@ -21,6 +21,8 @@ package org.ossreviewtoolkit.plugins.packagemanagers.bazel
 
 import java.io.File
 
+import org.apache.logging.log4j.kotlin.logger
```

Comment:
Commit message nits:
- But backticks around quoted file names
- "output*s* them"
- "the analy*sis* of the logs"